modules/mesos-cluster/main.tf

```diff
@@ -229,3 +229,15 @@ resource "aws_route53_record" "paas-mozilla-org-dns-zone" {
   ttl = 300
   records = ["${aws_elb.mozilla-org-elb.dns_name}"]
 }
+
+resource "aws_cloudwatch_metric_alarm" "cloudwatch-health-host-count-alarm" {
```

Comment:
To tell this alert which CloudWatch Metrics to check, we need to specify `namespace` and `dimensions`. The `HealthyHostCount` is a metric of the `AWS/ELB` namespace and we'd want the dimensions to be the `LoadBalancerName` of an ELB's healthy instances we want to monitor (in this case the mesos master one).